### PR TITLE
some fixes for vpn_openvpn_export.php

### PR DESCRIPTION
**Pull request by fabianfrz**

### CODE REVIEW

File: src/www/vpn_openvpn_export.php [hunk 1]
```diff
@@ -490,9 +490,9 @@
                         <?=gettext("Use a password to protect the pkcs12 file contents or key in Viscosity bundle.");?>
                       </div>
                       <div id="usepass_opts" style="display:none">
-                        <label ><?=gettext("Password :");?></label>
+                        <label for="pass"><?=gettext("Password:");?></label>
                         <input name="pass" id="pass" class="form-control" type="password" value="" />
-                        <label ><?=gettext("Confirm :");?></label>
+                        <label for="conf"><?=gettext("Confirm:");?></label>
```
Comment (fichtner) [inline src/www/vpn_openvpn_export.php]: I was going to suggest moving the colon outside of the issue. "Password" is readily translated, but "Confirm" is tricky. We should opt for "Confirmation".

File: src/www/vpn_openvpn_export.php [hunk 2]
```diff
@@ -686,7 +686,7 @@
                     <div class="hidden" for="help_for_clientpkg">
                       <br/>
                       <?= gettext("NOTES:") ?> <br/>
-                      <?= gettext("The &quot;XP&quot; Windows installers work on Windows XP and later versions. The &quot;win6&quot; Windows installers include a new tap-windows6 driver that works only on Windows Vista and later.") ?><br/>
+                      <?= gettext('The "XP" Windows installers work on Windows XP and later versions. The "win6" Windows installers include a new tap-windows6 driver that works only on Windows Vista and later.') ?><br/>
```
Comment (fichtner) [inline src/www/vpn_openvpn_export.php]: nice catch, html escaping isn't favourable for translation strings :)

File: src/www/vpn_openvpn_export.php (hunk 1 above)
Comment (fabianfrz) [inline src/www/vpn_openvpn_export.php]: or "Confirm Password:"

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**